.github/workflows/ci-deep.yml

```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
     inputs:
       fuzzRuns:
-        default: "100000"
+        default: "10000"
```

Comment:
why?

Reply:
@sogipec asked I believe because it is run very often and will take a lot of rpc calls